src/BuildScriptGenerator/IEnvironment.cs

```diff
@@ -14,14 +14,13 @@ namespace Microsoft.Oryx.BuildScriptGenerator
     public interface IEnvironment
     {
         /// <summary>
-        /// Gets values of an environment variable.
+        /// Gets the value of an environment variable.
         /// </summary>
-        /// <param name="name">Name of the environment variable. Is case-sensitive.</param>
+        /// <param name="name">Name of the environment variable. Case-sensitive.</param>
         /// <returns>
-        /// The value of the environment variable specified by variable,
-        /// or null if the environment variable is not found.
+        /// The value of the given environment variable, or null if the environment variable isn't found.
         /// </returns>
-        string GetEnvironmentVariable(string name);
+        string GetEnvironmentVariable(string name, string defaultValue = null);
```

Comment:
Why this change? I do not see it being used in any of the code you have in this PR

Reply:
I guess I used it somewhere and then changed the underlying code..
In any case, wouldn't hurt to have it